source/index.js

```diff
@@ -353,7 +354,7 @@ class ProgressBar {
 
 const htmlContent = `
 <!DOCTYPE html>
-<html lang={{REPLACE:LANG}}>
+<html {{REPLACE:LANG}}>
```

Comment:
@AndersonMamede I think "REPLACE:LANG" means to replace something with something else.
So I guess it doesn't fit.
If you create another property later, please also rename it to REPALCE:LANG. 😃